### PR TITLE
Updates Dependencies, Adds cli --suppress option

### PR DESCRIPTION
**Pull request by wayneashleyberry**
https://github.com/wayneashleyberry/phplint/issues/20

### CODE REVIEW

**Comment by Soullivaneuh** [comment]
:+1: 

Any stable release for this?

**Comment by wayneashleyberry** [comment]
Yeah, just published [v1.7.0](https://github.com/wayneashleyberry/phplint/releases/tag/v1.7.0] @Soullivaneuh

**Comment by wayneashleyberry** [comment]
```
npm i -g phplint@latest
```

File: index.js [hunk 1]
```diff
@@ -34,7 +34,16 @@ function iterate (filePaths, options, callback) {
     // If no cache then just lint
     if (!swap) {
       return lint(filePath, function (err, stdout, stderr) {
-        if (options.stdout) process.stdout.write(stdout)
+        if (options.stdout) {
+          if (options.suppress) {
+            if (stdout.substr(0, 28) !== 'No syntax errors detected in') {
```
Comment (Soullivaneuh) [inline index.js]: If forest. This could be optimized IMHO.

Comment (wayneashleyberry) [inline index.js]: Totally

**Pull request merged**